<launch>
    <!-- 定义carry_robot节点 -->
    <node name="w4a_complete_flow_node" pkg="carry_robot" type="w4a_complete_flow_node" output="screen">
        <!-- 定义抓取目标桌面的x坐标 -->
        <param name="grab_desk_x" value="1.90"/>
        <!-- 定义抓取目标桌面的y坐标 -->
        <param name="grab_desk_y" value="-1.83"/>

        <!-- 定义放置tag1的x坐标 -->
        <param name="tag_1_put_x" value="1.90"/>
        <!-- 定义放置tag1的y坐标 -->
        <param name="tag_1_put_y" value="-1.83"/>

        <!-- 定义放置tag2的x坐标 -->
        <param name="tag_2_put_x" value="1.6"/>
        <!-- 定义放置tag2的y坐标 -->
        <param name="tag_2_put_y" value="-0.77"/>

        <!-- 定义左侧放置偏移量 -->
        <param name="offset_left" value="0.1"/>
        <!-- 定义右侧放置偏移量 -->
        <param name="offset_right" value="0.1"/>
    </node>
</launch>
